backend/src/controllers/task.js

```diff
@@ -1,5 +1,5 @@
 const {Router} =  require('express');
-
+const Task = require('../schemas/taskschema')
```

Comment:
We needed to import the `Task` model that we use below to read / create / update the tasks in database.